src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -482,6 +482,15 @@ class PythonSetup(Subsystem):
         ),
         advanced=True,
     )
+    tailor_py_typed_targets = BoolOption(
+        default=False,
```

Comment:
I followed what we have set for other similar options and set it to `False`

Reply:
I think we may actually want to default to True?

Reply:
Alrighty then, done.